cmd/delete.go

```diff
@@ -26,7 +29,11 @@ var deleteCmd = &cobra.Command{ //nolint:gochecknoglobals
 		err := request.NewAPIClient(projectId, &apiKey).
 			DeleteIntegration(cmd.Context(), integrationId)
 		if err != nil {
-			logger.FatalErr("Unable to delete integration", err)
+			if errors.Is(err, clerk.ErrNoSessions) {
+				logger.FatalErr("Clerk session has expired, please log in using amp login", err)
```

Comment:
Clerk is an implementation detail that builders do not need to know about, can you revise this message and all the other error messages to "Session has expired, please log in again by running `amp login`"